ipn/ipnlocal/local.go

```diff
@@ -2470,6 +2457,19 @@ func (b *LocalBackend) Start(opts ipn.Options) error {
 	}
 	b.stateMachineLockedOnEntry(unlock)
 
+	// initialize Taildrive shares from saved state
+	fs, ok := b.sys.DriveForRemote.GetOK()
+	if ok {
```

Comment:
nit: maybe limit the `fs` scope?
```suggestion
	if fs, ok := b.sys.DriveForRemote.GetOK(); ok {
```